code/modules/cargo/packs.dm

```diff
@@ -1720,6 +1876,18 @@
 	for(var/i in 1 to 49)
 		new /mob/living/simple_animal/crab(.)
 
+/datum/supply_pack/critter/parrot
+	name = "Bird Crate"
+	desc = "Contains four expert telecommunication birds."
+	cost = 4000
+	contains = list(/mob/living/simple_animal/parrot)
+	crate_name = "parrot crate"
+
+/datum/supply_pack/critter/parrot/generate()
+	. = ..()
+	for(var/i in 1 to 4)
+		new /mob/living/simple_animal/parrot(.)
```

Comment:
you could add a small prob for the clockhawk since it's not available elsewhere in the game :)